### DISCUSSION TITLE
Debugging in a cshtml with razor engine

### DISCUSSION THREAD

**Discussion by Trgutylmaz** [adoconnection/RazorEngineCore#52] [Q&A] [answered]
Hi,

I want to debug a test.cshtml file while compiling with a razor engine so is there any support on the RazorEngineCore for .Net Core 3.1? In this way, I want to be able to detect errors in cshtml at runtime. 

![image](https://user-images.githubusercontent.com/53736865/114857753-50033a00-9df1-11eb-8d86-12b27f0ef31a.png)

**Comment by adoconnection** [comment]
Hi, im not sure it is possible

**Reply by shehrozeee** [reply, marked as answer]
I ran into an issue where I really needed the debugging and l was able to implement it.
you can find a working version here
[My Fork of this repo](https://github.com/shehrozeee/RazorEngineCore)
I have opened a pull request. if it gets merged by @adoconnection , you can use this repo other wise you can use mine.
